Content.Server/Body/Systems/InternalsSystem.cs

```diff
@@ -143,6 +143,11 @@ private void OnInhaleLocation(EntityUid uid, InternalsComponent component, Inhal
             // TODO: Should listen to gas tank updates instead I guess?
             _alerts.ShowAlert(uid, AlertType.Internals, GetSeverity(component));
         }
+        else if (component.Auto)
+        {
+            ToggleInternals(uid, uid, true, component);
+            component.Auto = false;
+        }
```

Comment:
i feel like it'd make more sense to just try and toggle the internals on mapinit rather than doing it like this.

Reply:
I'm not sure how mapinit works, when are on-mapinit functions called? After the entity is loaded?

Reply:
After it's first spawned but not on every load which is a much better place to do it.